pep-9999.rst

```diff
@@ -0,0 +1,317 @@
+   ============  =======================================================
+   libc6         libdl.so.2, libresolv.so.2, librt.so.1, libc.so.6,
+                 libpthread.so.0, libm.so.6, libutil.so.1, libnsl.so.1
+   libgcc1       libgcc_s.so.1
+   libgl1        libGL.so.1
+   libglib2.0-0  libgobject-2.0.so.0, libgthread-2.0.so.0, libglib-2.0.so.0
+   libice6       libICE.so.6
+   libsm6        libSM.so.6
+   libstdc++6    libstdc++.so.6
+   libx11-6      libX11.so.6
+   libxext6      libXext.so.6
+   libxrender1   libXrender.so.1
+   ============  =======================================================
+
+   On RPM-based systems, they are provided by these packages:
+
+   ============  =======================================================
+   Package       Libraries
+   ============  =======================================================
+   glib2         libglib-2.0.so.0, libgthread-2.0.so.0, libgobject-2.0.so.0
+   glibc         libresolv.so.2, libutil.so.1, libnsl.so.1, librt.so.1,
+                 libpthread.so.0, libdl.so.2, libm.so.6, libc.so.6
+   libICE        libICE.so.6
+   libX11        libX11.so.6
+   libXext:      libXext.so.6
+   libXrender    libXrender.so.1
+   libgcc:       libgcc_s.so.1
+   libstdc++     libstdc++.so.6
+   mesa          libGL.so.1
+   ============  =======================================================
+
+3. If the wheel contains binary executables or shared objects linked
+   against any allowed libraries that also export versioned symbols,
+   they may only depend on the following maximum versions::
+
+       GLIBC_2.17
+       CXXABI_1.3.7
+       GLIBCXX_3.4.19
+       GCC_4.8.5
+
+   As an example, ``manylinux2014`` wheels may include binary
+   artifacts that require ``glibc`` symbols at version ``GLIBC_2.12``,
+   because this an earlier version than the maximum of ``GLIBC_2.17``.
+4. If a wheel is built for any version of CPython 2 or CPython
+   versions 3.0 up to and including 3.2, it *must* include a CPython
+   ABI tag indicating its Unicode ABI.  A ``manylinux2014`` wheel
+   built against Python 2, then, must include either the ``cpy27mu``
+   tag indicating it was built against an interpreter with the UCS-4
+   ABI or the ``cpy27m`` tag indicating an interpreter with the UCS-2
+   ABI. [6]_ [7]_
+5. A wheel *must not* require the ``PyFPE_jbuf`` symbol.  This is
+   achieved by building it against a Python compiled *without* the
+   ``--with-fpectl`` ``configure`` flag.
+
+Compilation of Compliant Wheels
+===============================
+
+Like ``manylinux1``, the ``auditwheel`` tool adds ``manylinux2014``
+platform tags to ``linux`` wheels built by ``pip wheel`` or
+``bdist_wheel`` in a ``manylinux2014`` Docker container.
+
+Docker Images
+-------------
+
+A ``manylinux2014`` Docker image based on CentOS 7 x86_64 is provided
+for building binary ``linux`` wheels that can reliably be converted to
+``manylinux2014`` wheels. [8]_ This image comes with a full compiler
```

Comment:
There's nothing at the link given. Are there existing images somewhere else? Or should this say that the image will be provided?

Reply:
Yeah, my hope was that this might be done before the PEP was accepted, but since this now seems unlikely, I'll update it accordingly.